rollup.config.mjs

```diff
@@ -37,6 +37,8 @@ export default inputFiles.map((input) => {
 			esModule: false,
 			preserveModules: true,
 			validate: true,
+			banner:
+				'// NOTICE: This file is generated by Rollup. If you want to change,\n// please edit its ESM version file instead of this one.',
```

Comment:
`If you want to change` > `If you want to change it`

I am not sure about `ESM version file instead of this one.` maybe `ESM counterpart instead.`?

Reply:
From my perspective, I'm thinking of explaining how to rebuild it - what are our thoughts on this?

```suggestion
				'// NOTICE: This file is generated by Rollup. To modify it,\n// please instead edit the ESM counterpart and rebuild with Rollup (npm run build).',
```

Reply:
Thanks for the feedback. Updated with e80be6a6f.